src/components/Comments/ContainerComment.js

```diff
@@ -78,6 +78,12 @@ const ContainerComment = ({
       updateVote();
     }
   };
+  const handlePaddingBottom = () => {
+    if (!itemParent?.message || itemParent?.message === '') {
+      return calculatePaddingBtm() + calculationText(itemParent?.message).containerHeight;
+    }
+    return calculatePaddingBtm();
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic in the `handlePaddingBottom` function seems to be incorrect. It appears that you want to add the container height to the padding bottom when there is no message or the message is empty. However, your current condition will do this only when there is a message. You should reverse the condition.
```diff
-    if (!itemParent?.message || itemParent?.message === '') {
+    if (itemParent?.message && itemParent?.message !== '') {
         return calculatePaddingBtm() + calculationText(itemParent?.message).containerHeight;
     }
     return calculatePaddingBtm();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->